src/options.ts

```diff
@@ -236,6 +236,10 @@ export function getCommonCommandOpts() {
     '--match <name>',
     'run/push tests with a name or tags that matches a pattern'
   );
+  const namespace = createOption(
+    '--namespace <name>',
+    'Kibana namespace to use for the monitor. Defaults to the spaceId of the current project.'
```

Comment:
What is spaceId mean here? can you reword this properly? Kibana namespace -> datastream namespace as per your comment https://github.com/elastic/synthetics/pull/1021/files#r2122003292